### ISSUE TITLE
Panic exit after running generate

### ISSUE THREAD

**Issue by kacy** [steebchen/prisma-client-go#705]
Hi folks. We're currently using Prisma with our nodejs project, but our Go services use static queries that may drift in the future. We'd like to use the Prisma schema as a source of truth and integrate with the code generated from this library. However, can't seem to find a way around the panic error I'm receiving after using the same schema used in our node service.

The error I'm receiving: 

```
$ go run github.com/prisma/prisma-client-go generate
Prisma schema loaded from schema.prisma
Error:
                SpatialRefSys spatialRefSysActions
}
 from file client.gotpl: 230242:28: expected ';', found npx (and 91 more errors)
exit status 1



panic: could not run [generate]: exit status 1

goroutine 1 [running]:
main.main()
        /Users/me/Development/go/pkg/mod/github.com/prisma/prisma-client-go@v0.12.2/main.go:29 +0x40c
exit status 2
```

From what I can tell, the table doesn't actually matter. The error will always return the last table in the schema if I comment out things.

Not sure if it's helpful, but there are 63 tables and nearly 1000 lines of code in the schema. This is at the top of the schema if that helps with debugging:

```
generator client {
  output = "db_gen.go"
  package = "db"
  provider      = "go run github.com/prisma/prisma-client-go"
  // binaryTargets = ["native"]
  // provider      = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}
```

Thanks for taking a look!

**Comment by steebchen** [comment]
Sorry for the late response. Can you please provide your full schema? It seems to be related to code generation and that seems to be related to your schema.

**Comment by kacy** [comment]
Hey @steebchen. Thanks for the reply and totally understand given the holidays.

After reading your comment, I started looking for things that would cause a parsing error, and I found out the cause of the issue.

In our schema, we have a few `// comments` with inline backticks. The standard JS parser ignores it, but it breaks in this library. No worries though. Easy for us to fix in our configs.

Going to close the Issue. Thanks again for the hint and the work you all have put into Prisma.

**Comment by steebchen** [comment]
I'll re-open this as it's still a bug that should be fixed. Thanks for the helpful info though!

**Comment by steebchen** [comment]
@PetkoPetkov03 Can you please share the config part of your prisma schema and the commands you are running?

**Comment by PetkoPetkov03** [comment]
@steebchen I found the problem out of habit I used the wrong generator provider 

```
generator db {
  provider = "go run github.com/prisma/prisma-client-go"  // Here
}
```

**Comment by steebchen** [comment]
Closed in favor of #963 as it has a better description of the issue
